interface-definitions/load-balancing_reverse-proxy.xml.in

```diff
@@ -102,6 +102,39 @@
                   </leafNode>
                 </children>
               </node>
+              <node name="http-check">
+                <properties>
+                  <help>HTTP check configuration</help>
+                </properties>
+                <children>
+                  <leafNode name="method">
+                    <properties>
+                      <help>HTTP method used for health check</help>
+                      <valueHelp>
+                        <format>options|get|post|put</format>
+                        <description>HTTP method used for health checking</description>
+                      </valueHelp>
+                      <constraint>
+                        <regex>options|get|post|put</regex>
```

Comment:
What does “options” do? Is it correct method?

Reply:
HTTP OPTIONS request are used to query available request methods. haproxy uses that per default:

> An HTTP-layer health check sends an HTTP OPTIONS request to the server and expects to get a successful response. To enable it, add option httpchk to the backend section

It is used by default as OPTIONS requests should not trigger any intensive computation like a GET might (to render an HTML page for example).